include/library/spdm_return_status.h

```diff
@@ -0,0 +1,90 @@
+#ifndef SPDM_RETURN_STATUS_H
+#define SPDM_RETURN_STATUS_H
+
+typedef uint32_t libspdm_return_t;
+
+/* Returns 0 if status is LIBSPDM_STATUS_SUCCESS else it returns 1. */
```

Comment:
Would be nice to have a comment/diagram at the top that shows the bits of the return status, which bits correspond to what fields, and what is reserved/unused

Reply:
It is done.